### PR TITLE
delete webpack, go raw.

### PR DESCRIPTION
**Pull request by zerovox**
compiles things into the assets/game/ directory as expected. no more webpack dev server. `yarn:start` should still work, just spits the files on to the disk and has them served by the turbo-hearts server. It's using fancy shit like "sucrase" and "pax" (and "sass"?) instead of loaders and all that. I have no idea if it supports source-maps, I'm sure we could tweak it more to include those.

One weird thing, pax is a rust binary, but cargo.toml seem to be more about libraries, so might need to `cargo install pax` to globally install that thing. Would be nicer if it was packaged as an npm binary, or if cargo had some support for binary deps.

### CODE REVIEW

**Comment by tjwilson90** [comment]
claims it supports source maps https://github.com/nathan/pax#does-it-do-source-maps

**Comment by zerovox** [comment]
will test that with `sucrase --computeSourceMap true`

File: Cargo.toml [hunk 1]
```diff
@@ -25,6 +25,7 @@ warp = { version = "*", default-features = false }
 
 [dev-dependencies]
 tempfile = "*"
+pax = "*"
```
Comment (tjwilson90) [inline Cargo.toml]: yeah, I don't think you want to do this. I think you just want to `cargo install pax` in the build scripts.

Comment (tjwilson90) [reply Cargo.toml]: Might need to add the installed binary to PATH if it doesn't work automatically too.

**Pull request merged**